src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+
+
+public class ShootCommand extends Command {
+
+    private final Shooter shooter;
+    private final XboxController controller;
+
+    private double leftTrigger;
+
+    public ShootCommand(Shooter shooter, XboxController controller) {
+        // each subsystem used by the command must be passed into the
+        // addRequirements() method (which takes a vararg of Subsystem)
+
+        this.shooter = new Shooter();
+        this.controller = controller;
+        addRequirements();
+    }
+
+    /**
+     * The initial subroutine of a command.  Called once when the command is initially scheduled.
+     */
+    @Override
+    public void initialize() {
+        leftTrigger = controller.getRawAxis(XboxController.Axis.kLeftTrigger.value);
+    }
+
+    /**
+     * The main body of a command.  Called repeatedly while the command is scheduled.
+     * (That is, it is called repeatedly until {@link #isFinished()}) returns true.)
+     */
+    @Override
+    public void execute() {
+        this.shooter.runShooter(leftTrigger, leftTrigger);
```

Comment:
We dont need the right bottom shooter speed as it follows the top shooter. Also not 100% sure if this should be a variable input as there is no reason for the kitbot needing variable speed unless we test it and need it